### PR TITLE
[release 2.3] update org references to stolostron

### PR DESCRIPTION
**Pull request by SherinV**
Signed-off-by: Sherin Varughese <shvarugh@redhat.com>

Work in progress

### CODE REVIEW

File: Dockerfile
Comment (jlpadilla) [inline Dockerfile:4]: Should we update to go1.17 too?

File: config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml
Comment (jlpadilla) [inline config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml:16]: This image tag doesn't exist in the new org.  Just checking that we don't use it.  Maybe change it to a dummy value if it's only a sample.

File: vbh/build-harness-extensions/README.md
Comment (jlpadilla) [inline vbh/build-harness-extensions/README.md:14]: I believe `master` was renamed to `main`.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

Approving, but please review comments before merging.

File: config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml
Comment (jlpadilla) [reply config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml:16]: I believe this is why Travis failed.

File: Dockerfile
Comment (SherinV) [reply Dockerfile:4]: I have those changes in https://github.com/stolostron/search-operator/pull/94

Comment (jlpadilla) [reply Dockerfile:4]: Thanks, approved #94.

File: vbh/build-harness-extensions/README.md
Comment (SherinV) [reply vbh/build-harness-extensions/README.md]: Updated. Thanks.

File: config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml
Comment (SherinV) [reply config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml]: Updated to latest tag in stolostron.

**Pull request merged**
